Symfony/CS/Tokenizer/Token.php

```diff
@@ -99,9 +100,22 @@ public function equals($other)
         $selfPrototype = $this->getPrototype();
 
         foreach ($otherPrototype as $key => $val) {
-            if ($selfPrototype[$key] !== $val) {
+            // Make sure the token has such key
```

Comment:
I'm not trying to argue here, but comments and docblocks are already a bit inconsistent. There seems to be a preference for lowercase initial in comments, but there are plenty of examples with uppercase. Simfony's own code is littered with uppercase @param descriptions with no period.